Notes and Outlines/markdown modules/markdown.md

```diff
@@ -0,0 +1,302 @@
+
+    1. Item 1
+    2. Item 2
+    3. Item 3
+
+> 1. Item 1
+> 2. Item 2
+> 3. Item 3
+
+No matter what order you place the numbers in your plaintext file, markdown will generate the correct numbers in the displayed version.
+
+    10. Item 1
+    2. Item 2
+    9. Item 3
+
+This markup will still generate a  list that looks like this:
+
+> 10. Item 1
+> 2. Item 2
+> 9. Item 3
+
+#### List formatting
+
+Whether ordered or unordered, all lists follow some simple rules.
+
+You can embed sub-lists inside others by indenting the sub-lists.
+
+    * Item 1
+    * Item 2
+      * Sub-item 1
+      * Sub-item 2
+    * Item 3
+
+If you embed an ordered list inside another list, the numerical ordering will be unique for each sub-list.
+
+    1. Item 1
+    2. Item 2
+      1. Sub-item 1
+      2. Sub-item 2
+    3. Item 3
+      1. Sub-item 1
+      2. Sub-item 2
+
+Placing one or two newlines between list items will format the lists differently. Two newlines will space the items further part.
+
+    1. Item 1
+    2. Item 2
+
+> 1. Item 1
+> 2. Item 2
+
+    1. Item 1
+    
+    2. Item 2
+
+> 1. Item 1
+> 
+> 2. Item 2
+
+You can include multiple paragraphs inside of a list by indenting the paragraphs by two spaces.
+
+    1. Item 1, paragraph 1
+      
+      Item 1, paragraph 2
+    2. Item 2
+
+> 1. Item 1, paragraph 1
+>   
+>   Item 1, paragraph 2
+> 2. Item 2
+
+
+### Blockquotes
+
+Sometimes you want to quote a large chunk of text and put it off from the rest of the text. Blockquotes are a way to do this. To create a blockquote, add an greater-than sign at the beginning of the line of the paragraph you want to quote.
+
+Blockquotes can be nested by adding multiple greater-than signs.
+
+    Regular text.
+    
+    > A quoted paragraph.
+    > 
+    > Another quoted paragraph.
+    > 
+    > > Whoa nested blockquotes.
+    > 
+    > Final paragraph.
+
+> Regular text.
+> 
+> > A quoted paragraph.
+> > 
+> > Another quoted paragraph.
+> > 
+> > > Whoa nested blockquotes.
+> > 
+> > Final paragraph.
+
+Blockquotes can be used in a variety of ways. For instance, we're using blockquotes to show how code will look once formatted.
+
+### Code
+
+There are two ways to demarkate code in your text.
+
+Code will display in a monospaced font, and will escape any markdown formatting (so that you see the actual characters, not formatted text).
+
+#### In-line Code
+
+You can wrap a line of code in back-tic marks to make that line show up in a monospaced font.
+
+    To **bold text**, `add **two asterisks**`.
+
+> To **bold text**, `add **two asterisks**`.
+
+
+#### Code Blocks
+
+If you add 4 spaces at the beginning of a line, markdown will treat that whole line as code. Any spaces that you indent beyond the 4 will show up to the user, so that you can show embedded code blocks.
+
+        * List item 1
+        * List item 2
+          * List sub-item 1
+
+>     * List item 1
+>     * List item 2
+>       * List sub-item 1
+
```

Comment:
This is great! I just learned something! I've been using the back ticks for sharing large chunks of code. :smile: